languages/rust/oso/tests/test_polar_rust.rs

```diff
@@ -489,6 +489,7 @@ fn test_unify_externals() {
         if m == "cannot bind expression 'x = new Foo(1)' to '_instance_23'"));
 }
 
+#[ignore = "Re-enable after updating iterator implementation"]
```

Comment:
Does this need to be re-enabled?

Reply:
After the Rust changes are implemented (follow up PR incoming)